src/options.c

```diff
@@ -98,6 +98,13 @@ static void usage(const char *argv0, int ret) {
 	    "--animation-clamping\n"
 	    "  Whether to clamp animations (default: true)\n"
 	    "\n"
+	    "--animation-opacity-min (default: 0.0).\n"
+	    "  Minim Opacity of animation processing.\n"
```

Comment:
Bad english -- a fixed version would be "Minimum opacity of animation processing".
But I feel that that's a bit imprecise. How about "Minimum opacity for pixmap blending during window animation" instead?